CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ to know that you have a clean slate: `bundle && rake`
 require no new tests. If you are adding functionality or fixing a bug, we need
 a test!
 
-4. Make the test pass.
+4. Make the test pass. `rvm use 1.9.3 && rake`​ and ​`rvm use 2.2.3 && rake`​
```

Comment:
Some folks may be using `rbenv` or another service, could we also be more general in our description of why?  Maybe an example for rvm in parenthesis